src/README.md

```diff
@@ -0,0 +1,50 @@
+<!-- This README file is going to be the one displayed on the Grafana.com website for your plugin. Uncomment and replace the content here before publishing.
+
+Remove any remaining comments before publishing as these may be displayed on Grafana.com -->
```

Comment:
do we need to do this?

Reply:
I think so, haven't looked into the packaging/publishing part of this yet was just trying to get the code complete first

Reply:
done